libff/algebra/fields/tests/test_fields.cpp

```diff
@@ -369,12 +370,19 @@ void do_test_signed_digits(const FieldT &value, const size_t digit_size)
     //   c = digit size and
     //   d_i = i-th signed digit of size c
     FieldT accum = FieldT::zero();
+
     for (size_t i = 0; i < num_digits; ++i) {
         accum = accum * FieldT((long)1 << digit_size);
         ssize_t digit =
             field_get_signed_digit(v, digit_size, num_digits - 1 - i);
         accum = accum + FieldT((long)digit);
 
+        printf(
```

Comment:
let's remove this for now to avoid cluttering the test output.

Reply:
Done.